### ISSUE TITLE
Iomad sign-up local plugin does not work unless "force email to be username" is set to "no".

### ISSUE THREAD

**Issue by encodecreate** [iomad/iomad#1729]
When going to a company's custom login page, if "force email to be username" is set to "yes", the sign-up screen will include a username field but not an email field.

The user cannot submit the signup form, but instead receives a "missing required fields" error.

If "force email to be username" is set to "no," then username and email fields are both present and the user signup experience works as expected.

**Comment by turf212** [comment]
is this using the /login/signup.php page?

What version of IOMAD?

**Comment by encodecreate** [comment]
Yes, /login/signup.php

Latest version of Iomad.

**Comment by knk-danielgavriliuc** [comment]
is there a workaround for this? i have the same problem.

```
Notice: Undefined index: email in /var/www/html/my-iomad.com/login/signup_form.php on line 186

Notice: Undefined index: email in /var/www/html/my-iomad.com/lib/authlib.php on line 1074

Notice: Undefined index: email in /var/www/html/my-iomad.com/lib/authlib.php on line 1120
```

**Comment by turf212** [comment]
closed in 1a984b71ce1264b28cec02dd463fb28f7e14edb7 and 749ad1dcd332a142c064677a20f20ecbb03a5b61
